lib/snaplet/Taskfile.yaml

```diff
@@ -8,10 +8,23 @@ includes:
 vars:
   SNAPSHOT_NAME:
     sh: date +%Y-%m-%d # YYYY-MM-DD
-  SNAPSHOT_PATH: ".snaplet/snapshots/{{.SNAPSHOT_NAME}}"
-  SNAPSHOT_ENV: dev
+  SNAPSHOT_ENV: '{{.SNAPSHOT_ENV | default "dev"}}'
+  SNAPSHOT_PATH: .snaplet/snapshots/{{.SNAPSHOT_ENV}}/{{.SNAPSHOT_NAME}}
 
 tasks:
+  validate-copycat-secret:
+    desc: Validates that the `COPYCAT_SECRET` env var has been passed.
+    silent: true
+    internal: true
+    cmds:
+      - |
+        if [[ -z "{{.COPYCAT_SECRET}}" ]]; then
```

Comment:
this `{{.name}}` syntax is how you reference env variables in taskfile?

Reply:
Yeah, more info here: https://taskfile.dev/usage/#variables